### ISSUE TITLE
[a11y] - missing landmark for attribution / copyright section

### ISSUE THREAD

**Issue by transkamp** [open-pioneer/trails-openlayers-base-packages#448]
### Issue description

open https://open-pioneer.github.io/trails-demo/openlayers-base-packages/dev/samples/test-toc/

for a11y reasons it is better to markup the attribution section with `role="region"` and `aria-label="Attributionen/ Copyright"`
https://www.w3.org/WAI/ARIA/apg/patterns/landmarks/examples/HTML5.html

![Image](https://github.com/user-attachments/assets/5573ebed-9fde-4cd9-b160-27c60628b3d4)

### Validations

- [x] Read and follow our [code of conduct](https://github.com/open-pioneer/.github/blob/main/CODE_OF_CONDUCT.md)
- [x] Read the [contributing guidelines](https://github.com/open-pioneer/trails-starter/blob/main/docs/Contributing.md)
- [x] Check that there isn't already a similar issue
